entc/gen/template/migrate/schema.tmpl

```diff
@@ -129,6 +129,9 @@ func init() {
 				{{- with $ant.Options }}
 					Options: "{{ . }}",
 				{{- end }}
+				{{- with $ant.Check }}
+					Check: "{{ . }}",
+				{{- end }}
```

Comment:
This should be removed from the table's annotation and should be added to the column's annotation.